.github/workflows/qa.yml

```
@@ -1,6 +1,6 @@
 name: QA
 
-on: push
+on: [push, pull_request, workflow_dispatch]
```

```suggestion
on: [push, pull_request]
```

I can see the value of having this on PRs, but we don't need the manual dispatch.